cuegui/cuegui/FrameMonitorTree.py

```diff
@@ -73,7 +68,7 @@ def __init__(self, parent):
                        data=lambda job, frame: frame.data.layer_name,
                        tip="The layer that the frame is in.")
         self.addColumn("Status", 100, id=4,
-                       data=lambda job, frame: str(frame.data.state),
+                       data=lambda job, frame: self.getStateString(frame.data.state),
```

Comment:
Can we use `job_pb2.FrameState.Name()` instead?

Reply:
Thanks! I thought there was an easier way to get the names.